src/stack/Stack.java

```diff
@@ -19,38 +29,45 @@ public Stack(int size) {
 	 * Add element into stack
 	 */
 	public void push(int value) {
-
+		if (tos == data.length) {
+			ensureCapacity();
+		}
+		data[++tos] = value;
 	}
 
 	/**
 	 * Get element from the stack
-	 * @return
 	 */
 	public int pop() {
-
-		return 0;
+		if (isEmpty()){
+			System.out.println("Empty");
```

Comment:
If the stack is empty, calling pop () should call EmptyStackException, which you do not have